docs/invoice-spec.md

```diff
@@ -144,7 +144,10 @@ Between groups and conditions, it is possible to build tree-like dependency stru
 
 By default, if no condition is provided, an item is a member of the "global" group, and is required.
 
-- `memberOf`: A list of groups that this parcel is a member of. When a `memberOf` clause is present, the parcel is removed from the default global group and placed into _just_ the groups listed in the `memberOf` clause. `memberOf = []` indicates that this parcel is a member of no groups (including the global group). It is an error if a parcel references a group that is undefined in the `[[group]]` list. (OPTIONAL)
+- `memberOf`: A list of groups that this parcel is a member of. When a `memberOf` clause is present, the parcel is removed from the default global group and placed into _just_ the groups listed in the `memberOf` clause.
```

Comment:
Possibly picky (since you cover this in the very next paragraph) but may be worth writing:

```suggestion
- `memberOf`: A list of groups that this parcel is a member of. When a nonempty `memberOf` clause is present, the parcel is removed from the default global group and placed into _just_ the groups listed in the `memberOf` clause.
```